README.md

```diff
@@ -32,7 +32,7 @@ at elfner@itp.uni-frankfurt.de.
 
 SMASH is known to compile and work with one of these compilers (which have the
 required C++11 features):
-- gcc >= 4.8
+- gcc >= 5.0
```

Comment:
Should we mention somewhere that, even if not usable to run the tests, gcc 4.8 should be still sufficient to compile and run the program for most uses?

Reply:
It will be (I am about adding it) in the release note of the "only" 2.2 release (in the GitHub sense). I believe it is enough.

Reply:
I meant [the release notes](https://github.com/smash-transport/smash/releases/tag/SMASH-2.2).

About dropping the singularity container definition, this should be a fix of the CHANGELOG section about the 2.2, right? If you think we should/can put it here, is a formulation like _The singularity container definition has been dropped, since a singularity container can be obtained from the shipped docker_ in the **Removed** section of `SMASH-2.2` 🆗?

Reply:
Yes, thank you.